src/components/PostGrid.jsx

```diff
@@ -14,62 +13,88 @@ function PostGrid({ data }) {
     option === "issues" ? window.open(`${repoLink}/issues`) : window.open(repoLink);
   };
 
+  const handleRedirect = (contributor) => {
+    // window.open(`https://github.com/${contributor}`);
+  };
+
   return (
-    <div className=" bg-offWhite rounded-xl p-6 font-roboto cursor-pointer ">
-      {/* Avator Container */}
-      <div className=" w-full flex  mb-3 ">
-        <div className="bg-blue-400 w-10 h-10 overflow-hidden  rounded-full mr-3 ">
-          <img
-            className="object-cover"
-            src={getAvatar(data?.contributors[0])}
-            alt="Avatar 01"
-            width={500}
-            height={500}
-          />
+    <div className=" bg-offWhite rounded-xl p-6 font-roboto cursor-pointer flex flex-col justify-between ">
+      <div className="flex flex-col">
+        {/* Avator Container */}
+        <div className=" w-full flex ">
+          <div className="bg-blue-400 w-10 h-10 overflow-hidden  rounded-full mr-3 ">
+            <img
+              className="object-cover"
+              src={getAvatar(data?.contributors[0])}
+              alt="Avatar 01"
+              width={500}
+              height={500}
+              onClick={handleRedirect(data?.contributors[0])}
+            />
+          </div>
+          <div className="bg-blue-400 w-10 h-10 overflow-hidden  rounded-full mr-3 ">
+            <img
+              className="object-cover"
+              src={getAvatar(data?.contributors[1])}
+              alt="Avatar 02"
+              width={500}
+              height={500}
+              onClick={handleRedirect(data?.contributors[1])}
+            />
+          </div>
+        </div>
+        {/* Title */}
+        <div
+          className=" text-grey text-sm font-medium mb-1 h-14 overflow-hidden cursor-pointer flex items-center"
+          onClick={handleClick}
+        >
+          <h1>{data.repo_name} </h1>
         </div>
-        <div className="bg-blue-400 w-10 h-10 overflow-hidden  rounded-full mr-3 ">
+        {/* Description */}
+        <div className=" text-lightGrey text-sm mb-2 ">
+          <h3> {truncate(data.description)} </h3>{" "}
+        </div>
+        {/* Cover photo */}
+        <div className="w-full bg-transparent h-28 overflow-hidden rounded-md mb-2 flex justify-center">
           <img
-            className="object-cover"
-            src={getAvatar(data?.contributors[1])}
-            alt="Avatar 02"
-            width={500}
-            height={500}
+            className="object-contain h-full"
+            src={`https://opengraph.githubassets.com/1/${data.repo_name}`}
```

Comment:
This is perfect

Reply:
I will remove the stars and issues favicons and also put the upvote button at the top right corner of the card